input/pagecontent/index.md

```diff
@@ -125,8 +125,8 @@ This guide is divided into several pages which are listed at the top of each pag
   - [Profiles and Extensions](profiles-and-extensions.html): This page describes the profiles and extensions that are defined in this guide to exchange quality data. Each profile page includes a narrative description and guidance, formal definition and a "Quick Start" guide which summarises the supported search transactions for each profile. Although the guidance typically focuses on the profiled elements, it may also may focus on un-profiled elements to aid with implementation.
```

Comment:
remove the word "quality" from "exchange quality"

Reply:
Removed, thanks.

Reply:
Can't see this change.